melodies_monet/driver.py

```diff
@@ -211,6 +243,11 @@ def mask_and_scale(self):
                             self.obj[v].data += scale
                         elif d['unit_scale_method'] == '-':
                             self.obj[v].data += -1 * scale
+                    if 'obs_unit' in d:
```

Comment:
This is a good question. I think Jian added this as an option independent of the typical more generic scaling. But I wonder if we want to make it elif with a warning if you provide both options, so that users do not use both options and get odd results. @jianheACM can you comment on this and provide Colin some thoughts on what to add into the docs for this?